README.md

```diff
@@ -39,17 +32,22 @@ There is a new Max. resolution select field in the inputstream.adaptive settings
 Auto will select the best resolution matching to your videoplayer display rect without any limits.
 If your display resolution is 720p, you will not be able to watch 1080p videos if there are video representations available closer to 720p.
 
-
-##### TODO's:
-- Adaptive bitrate switching is prepared but currently not yet activated
+#### TODO's:
+- Adaptive bitrate switching has been enabled on Kodi 20 but need more improvements
```

Comment:
*needs

Reply:
_Adaptive bitrate switching has been implemented in Kodi as of v20, but will continue to be developed over time._